### DISCUSSION TITLE
“Automate the Boring Stuff with Python Programming” FREE?!

### DISCUSSION THREAD

**Discussion by iam-hector** [Fairchild-Coding-Club/START-HERE#10] [General]
Al Sweigart author of “Automate the Boring Stuff with Python Programming” is offering a course on the subject on Udemy for FREE!! For a limited time, just use  code “APR2021FREE” at checkout!  https://www.udemy.com/course/automate/

**Comment by TzZek** [comment]
Also check out DigitalU, gives us lots of Udemy courses for free! Machine learning, AI, Data Science and TONS of other courses.
